app/handlers/callback.py

```diff
@@ -68,13 +68,17 @@ def execute_put(self, *args, **kwargs):
     def _post(self, *args, **kwargs):
         try:
             lab_name = self.get_query_argument(models.LAB_NAME_KEY)
+            status_string = self.get_argument(models.LAVA_STATUS_STR_KEY)
```

Comment:
This is LAVA-specific so it shouldn't be done in the main `CallbackHandler` class.  It should instead be done in `LavaCallbackHandler` or in the task handling the callback.

Reply:
moved handling code to lava.py